diff --git a/dist/components/Autocomplete/Autocomplete.svelte b/dist/components/Autocomplete/Autocomplete.svelte
index 85a6fd1e94f108d11d5c61c0cc125fc630d86e6a..00567f19b7677bd90a28223a1beb6d0b740be8f0 100644
--- a/dist/components/Autocomplete/Autocomplete.svelte
+++ b/dist/components/Autocomplete/Autocomplete.svelte
@@ -85,7 +85,7 @@ $:
 						out:dynamicTransition|local={{ transition: transitionOut, params: transitionOutParams, enabled: transitions }}
 					>
 						<button class="autocomplete-button {classesButton}" type="button" on:click={() => onSelection(option)} on:click on:keypress>
-							{@html option.label}
+							{option.label}
 						</button>
 					</li>
 				{/each}