diff --git a/src/core/sync/write_tsconfig.js b/src/core/sync/write_tsconfig.js
index 9f3a083649fe8645380e9a3558c6bef5c44530ff..cbda8cdc76ba32282c2975a60d2adf70dedce05a 100644
--- a/src/core/sync/write_tsconfig.js
+++ b/src/core/sync/write_tsconfig.js
@@ -88,7 +88,8 @@ export function get_tsconfig(kit, include_base_url) {
 	const include = new Set([
 		'ambient.d.ts',
 		'./types/**/$types.d.ts',
-		config_relative('vite.config.ts')
+		config_relative('vite.config.ts'),
+		config_relative('tailwind.config.ts')
 	]);
 	// TODO(v2): find a better way to include all src files. We can't just use routes/lib only because
 	// people might have other folders/files in src that they want included.
